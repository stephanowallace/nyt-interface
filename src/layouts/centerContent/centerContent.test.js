import React from 'react';
import { shallow } from 'enzyme';
import CenterContent from '.';

describe('CenterContent', () => {
  it('Renders default state', () => {
    const component = shallow(<CenterContent><div>content</div></CenterContent>);
    expect(component).toMatchSnapshot();
  });
});
