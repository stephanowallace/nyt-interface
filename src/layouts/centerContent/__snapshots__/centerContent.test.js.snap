// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CenterContent Renders default state 1`] = `
<Fragment>
  <div
    className="body"
  >
    <div>
      content
    </div>
  </div>
</Fragment>
`;
